src/components/CollapsibleSection.js

```diff
@@ -62,7 +62,10 @@ export class CollapsibleSection extends Component {
 }
 
 CollapsibleSection.propTypes = {
-  children: PropTypes.element.isRequired,
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
```

Comment:
I forget; is this maybe equivalent to `PropTypes.element`?

Reply:
Yeah I guess so.. pushed up that change.